### DISCUSSION TITLE
Remove Consumable from User

### DISCUSSION THREAD

**Discussion by JeffTMRS** [snipe/snipe-it#12666] [Q&A]
Is this possible? I want to start tracking our Toners and checked one out to myself as a test, I am not sure how to remove it from my account now:

![image](https://user-images.githubusercontent.com/127872527/225051318-2ad3fb47-8bdb-4544-9e6c-a46d9e103c0a.png)

**Comment by qay21** [comment]
Hi @JeffTMRS 

As I understand the underlying logic, consumables are not meant to be checked in and out between various users. By nature, a consumable is supposedly destroyed when used. The "consumables" feature of SnipeIT is probably more thought as a date and volume tracking of checkouts. The "Oh, this user received a new Toner last week, why are they requesting a new one today ? That sound strange" kind of reminders.

That being said, I'm unsure of why you can only checkout consumables to Users. Checking out consumables to Assets (eg. a printer) would probably a good idea too.

**Reply by SertITN** [reply]
There is a problem with some Brother toners. Once inserted into the printer "burn" the chip in such a way that if they are removed and reinserted they are no longer recognized by the printer. There was a bug in some models, solved.

**Reply by qay21** [reply]
I think you got lost here sir
